### PR TITLE
feat/view parcours

### PR DESCRIPTION
**Pull request by SilGosker**

### CODE REVIEW

File: Kbs.Business.Tests/Parcours/CourseDifficultyExtensionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using Kbs.Business.Course;
+
+namespace Kbs.Business.Parcours;
```
Comment (turtleman31) [inline Kbs.Business.Tests/Parcours/CourseDifficultyExtensionsTests.cs:3]: change to Course

File: Kbs.Business.Tests/Parcours/CourseEntityTests.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using Kbs.Business.Course;
+
+namespace Kbs.Business.Parcours;
```
Comment (turtleman31) [inline Kbs.Business.Tests/Parcours/CourseEntityTests.cs:3]: change to Course

File: Kbs.Data/Course/CourseRepository.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿using Dapper;
+using Kbs.Business.Course;
+using Microsoft.Data.SqlClient;
+
+namespace Kbs.Data.Parcours;
```
Comment (turtleman31) [inline Kbs.Data/Course/CourseRepository.cs:5]: change to Cours

File: Kbs.Wpf/Course/Read/Details/ReadCourseDetailsPage.xaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+﻿<Page x:Class="Kbs.Wpf.Course.Read.Details.ReadCourseDetailsPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:details="clr-namespace:Kbs.Wpf.Course.Read.Details"
+      mc:Ignorable="d" 
+      d:DesignHeight="600" d:DesignWidth="900"
+      Title="ReadParcoursDetailsPage">
```
Comment (turtleman31) [inline Kbs.Wpf/Course/Read/Details/ReadCourseDetailsPage.xaml:9]: change to Course

File: Kbs.Wpf/Course/Read/Index/ReadCourseIndexPage.xaml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+﻿<Page x:Class="Kbs.Wpf.Course.Read.Index.ReadCourseIndexPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:index="clr-namespace:Kbs.Wpf.Course.Read.Index"
+      mc:Ignorable="d"
+      d:DesignHeight="1200" d:DesignWidth="900"
+      Title="ReadParcoursIndexPage">
```
Comment (turtleman31) [inline Kbs.Wpf/Course/Read/Index/ReadCourseIndexPage.xaml:9]: change to Course

**Review by Katanax28 — APPROVED** [review]

**Pull request merged**